src/vm/threadsuspend.cpp

```diff
@@ -8342,7 +8342,7 @@ void ThreadSuspend::SuspendEE(SUSPEND_REASON reason)
 #endif //TIME_SUSPEND
 }
 
-#if defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
+#if !defined(_TARGET_ARM64) && defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
```

Comment:
In addition, I think th define is incorrect (note the missing trailing _ or incorrect leading _)